IncludeToolbox/Formatter/IncludeFormatter.cs

```diff
@@ -198,6 +206,13 @@ private static List<IncludeLineInfo> SortIncludes(IList<IncludeLineInfo> lines,
                     outSortedList.Add(sortedLine);
                     firstLine = false;
                 }
+
```

Comment:
Not sure if this was a problem before, but now that there may be fewer includes than we had originally, there can be non-include content past-the-end of the sorted includes. This was exposed by the unit tests.